docs/installation/index.md

```diff
@@ -9,4 +9,7 @@ Die Erstinstallation von evcc erfordert aktuell noch etwas Fachwissen. Du sollte
 - [Debian, Ubuntu, Raspberry Pi OS](./installation/linux)
 - [macOS](./installation/macos)
 - [Docker, Synology](./installation/docker)
+- [Home Assistant addon](./installation/home-assistant)
+- [Proxmox](./installation/proxmox)
 - [Manuelle Installation (inkl. Windows)](./installation/manual)
+- [Konfiguration (manuell)](./installation/configuration)
```

Comment:
Reihenfolge: Home Assistant und Proxmox sollten hier an letzter Stelle stehen, da community-supported.

Generell bin ich immer noch nicht ganz überzeugt davon, dass die Beiden so prominent in den Installationsbereich (auch linke Navi) gehören. Andererseits fällt mich auch nicht viel Schöneres ein, da das ja schon der Ort ist, bei dem neue Nutzer starten.

Daher erstmal die Reihenfolge.